daemon/src/projection.rs

```diff
@@ -329,7 +329,7 @@ impl Cfd {
             CollaborativeSettlementProposalAccepted => {
                 self.pending_settlement_proposal_price = None;
 
-                self.state = CfdState::IncomingSettlementProposal;
+                self.state = CfdState::PendingClose;
```

Comment:
I'm not sure whether this is strictly correct - it would be better to only put into `PendingClose` state after the protocol has run. Otherwise, if we try to accept an "old" proposal, the taker might have moved on with their cfd and e.g. force-closed it.

Reply:
This is a pretty "short" state anyway because we will either get "Completed" or "Failed" afterwards. Not sure if it is worth to introduce a completely new state.

Reply:
fair enough. It certainly is less of a concern than still displaying the action....